src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidator.java

```diff
@@ -131,13 +133,14 @@ private boolean validateType(String type, Errors errors, String loggingContext)
         return true;
     }
 
-    private boolean validateDateBecameInterestedPerson(LocalDate dateBecameInterestedPerson, Errors errors,
+    private boolean validateDateBecameInterestedPerson(LocalDate dateBecameInterestedPerson, LocalDate trustCreationDate, Errors errors,
                                                        String loggingContext) {
         String qualifiedFieldName = getQualifiedFieldName(PARENT_FIELD,
                 TrustCorporateDto.DATE_BECAME_INTERESTED_PERSON_FIELD);
 
         return UtilsValidators.isNotNull(dateBecameInterestedPerson, qualifiedFieldName, errors, loggingContext)
-                && DateValidators.isDateInPast(dateBecameInterestedPerson, qualifiedFieldName, errors, loggingContext);
+                && DateValidators.isDateInPast(dateBecameInterestedPerson, qualifiedFieldName, errors, loggingContext)
+                && DateValidators.isDateBecameInterestedPersonOnOrAfterCreationDate(dateBecameInterestedPerson, trustCreationDate, qualifiedFieldName, errors, loggingContext);
```

Comment:
I'm wondering which validation AC in [this](https://companieshouse.atlassian.net/browse/UAR-1452) story the new `isDateBecameInterestedPersonOnOrAfterCreationDate` check relates to?

Reply:
Just spoke to Vernon and he has agreed that we do need to check the ceased date against this date as well and has updated https://companieshouse.atlassian.net/browse/UAR-1452 that will relate to Acceptance Criteria 7 on invalid dates.

Reply:
👍